tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -528,48 +571,86 @@ def deployments_to_revert(self):
+                        "Skipping adding the [client.admin] details present in rook-ceph-mons-keyring"
+                        "as the secret details are already fecthed with rook-ceph-admin-keyring"
+                    )
+                    break
+                key = None
+                caps = []
+                if block.startswith("key ="):
+                    key = block.split(" = ")[1].strip()
+                elif "caps" in block:
+                    caps.append(block.strip())
+                if key:
+                    logger.info(f"Found key :{key}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)